File: backend/models/HinhAnhSanPham.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const HinhAnhSanPham = sequelize.define(
  "HinhAnhSanPham",
  {
    id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
    san_pham_id: { type: DataTypes.INTEGER, allowNull: false },
    bien_the_id: { type: DataTypes.INTEGER, allowNull: true },
    url_anh: { type: DataTypes.STRING(255), allowNull: false },
    alt_text: { type: DataTypes.STRING(255) },
    la_anh_chinh: { type: DataTypes.BOOLEAN, defaultValue: false },
    thu_tu: { type: DataTypes.INTEGER, defaultValue: 0 },
  },
  {
    tableName: "HinhAnhSanPham",
    timestamps: false,
  },
);

module.exports = HinhAnhSanPham;

File: backend-csharp/Models/HinhAnhSanPham.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class HinhAnhSanPham
{
    public long Id { get; set; }

    public long SanPhamId { get; set; }

    public long? BienTheId { get; set; }

    public string UrlAnh { get; set; } = null!;

    public string? AltText { get; set; }

    public bool LaAnhChinh { get; set; }

    public int ThuTu { get; set; }

    public virtual BienTheSanPham? BienThe { get; set; }

    public virtual SanPham SanPham { get; set; } = null!;
}
